stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -1130,7 +1141,7 @@ extension Unsafe${Mutable}RawBufferPointer {
 %  if Mutable:
   @inlinable
   @_alwaysEmitIntoClient
-  public func withContiguousMutableStorageIfAvailable<R>(
+  public func withContiguousMutableStorageIfAvailable<R: ~Copyable>(
```

Comment:
Should we make a `withContiguousMutableStorageIfAvailable()` for non-copyable elements if we haven't done so for the `MutableCollection` symbol it overrides?